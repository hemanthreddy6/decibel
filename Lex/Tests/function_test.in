main <- (a: int, b: int): float {
    a: int <- 16#09#2024;
    b: float <- a+2;
    return b;
}

main {
    
}