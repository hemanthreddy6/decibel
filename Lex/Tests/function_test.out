MAIN
<-
(
IDENTIFIER
:
INT
,
IDENTIFIER
:
INT
)
:
FLOAT
{
IDENTIFIER
:
INT
<-
INT_LITERAL
INVALID_SYMBOL
INT_LITERAL
INVALID_SYMBOL
INT_LITERAL
;
IDENTIFIER
:
FLOAT
<-
IDENTIFIER
+
INT_LITERAL
;
RETURN
IDENTIFIER
;
}
MAIN
{
}
